nginx/common.sls

```diff
@@ -1,7 +1,7 @@
 {% from "nginx/map.jinja" import nginx as nginx_map with context %}
 {% set nginx = pillar.get('nginx', {}) -%}
-{% set home = nginx.get('home', '/var/www') -%}
-{% set conf_dir = nginx.get('conf_dir', '/etc/nginx') -%}
+{% set home = nginx.get('home', nginx_map.home) -%}
```

Comment:
@bmwiedemann the '/var/www' are the defaults if ```home``` isn't set in map.jinja or provided by pillar.

same as the conf_dir settings...

so this commit seems unnecesary, the other two commits are fixing your problem just fine.

Reply:
I had a "/srv/www" in map.jinja and it was not used without this change. Pillar was empty, though.

Reply:
I see the issue see:
``` {% set nginx = pillar.get('nginx', {}) -%}```

this should be:
``` {% set nginx = pillar.get('nginx_map', {}) -%}```

Reply:
but the whole map.jinja and pillar.get is a bit iffy with the nginx.sls state

why export nginx in map.jinja and then import is as nginx_map...and not use it,
the set commands all drag settings from pillar under the ```nginx``` key...